GUI/Model/ModSearch.cs

```diff
@@ -32,9 +32,11 @@ public sealed class ModSearch
             bool? upgradeable, bool? replaceable,
             string combined = null)
         {
-            Name         = CkanModule.nonAlphaNums.Replace(byName, "");
+            Name = (ShouldNegateTerm(byName, out string subName) ? "-" : "")
+                + CkanModule.nonAlphaNums.Replace(subName, "");
             initStringList(Authors, byAuthors);
-            Description  = CkanModule.nonAlphaNums.Replace(byDescription, "");
+            Description = (ShouldNegateTerm(byDescription, out string subDesc) ? "-" : "")
+                + CkanModule.nonAlphaNums.Replace(byDescription, "");
```

Comment:
While the regex replacement saves us here, I think this should technically be `CkanModule.nonAlphaNums.Replace(subDesc, "");

Reply:
Good catch! That was indeed the point of using `out string` there.